reporthandling/apis/statusinfo.go

```diff
@@ -23,12 +23,7 @@ func (s *StatusInfo) IsFailed() bool {
 	return s.InnerStatus == StatusFailed
 }
 
-// IsExcluded is the status excluded
-func (s *StatusInfo) IsExcluded() bool {
-	return s.InnerStatus == StatusExcluded
-}
-
 // IsSkipped is the status skipped
 func (s *StatusInfo) IsSkipped() bool {
-	return !s.IsExcluded() && !s.IsPassed() && !s.IsFailed()
+	return !s.IsPassed() && !s.IsFailed() && s.InnerStatus != StatusUnknown
```

Comment:
[Q] `s.InnerStatus != StatusUnknown` Why? please add a comment with the use case